### ISSUE TITLE
[css-fonts] which generic in a font-family list influences system fallback?

### ISSUE THREAD

**Issue by heycam** [w3c/csswg-drafts#4562]
Breaking this out from https://github.com/w3c/csswg-drafts/issues/4442:

@heycam:
> Do we need to add wording about which generic, if multiple are listed, influence any implementation specific system fallback?

@jfkthame:
> Presumably the first generic mentioned should be the one to influence fallback after we fall off the end of the font-family list.

Is this something we need to have a SHOULD-level statement for in the spec?

**Comment by svgeesus** [comment]
Yes, we need to add that wording. I agree that the first generic mentioned (regardless of whether it matched to a real font or not) is the best option, because the font stack is a priority ordered list so the first generic should most closely correspond to authorial intent.

**Comment by litherum** [comment]
> the first generic method should be the one to influence fallback

I disagreed in https://github.com/w3c/csswg-drafts/issues/4442#issuecomment-547577857
